content/guides/models/registry/configure_registry.md

```diff
@@ -9,6 +9,53 @@ weight: 3
 
 A registry admin can [configure registry roles]({{< relref "configure_registry.md#configure-registry-roles" >}}), [add users]({{< relref "configure_registry.md#add-a-user-or-a-team-to-a-registry" >}}), or [remove users]({{< relref "configure_registry.md#remove-a-user-or-team-from-a-registry" >}}) from a registry by configuring the registry's settings.
 
+## Diagram
+
+This diagram illustrates the hierarchical permission structure in Weights & Biases (W&B), showing the relationships between Organization, Team, and Registry roles and how permissions are inherited. Users receive the highest permission level between their individual assignment and team membership, with Registry roles limited to three fixed types (Admin, Member, Viewer) that determine what actions users can perform. 
```

Comment:
I don't think we need the "...Weights & Biases (W&B)..." bit since we use "W&B" (almost) exclusively throughout the docs.

Note: This is something that was brought forward by a founding PM and marketing team at the time. So unsure if the branding has changed since then.

```suggestion
This diagram illustrates the hierarchical permission structure in W&B, showing the relationships between Organization, Team, and Registry roles and how permissions are inherited. Users receive the highest permission level between their individual assignment and team membership, with Registry roles limited to three fixed types (Admin, Member, Viewer) that determine what actions users can perform. 
```